src/ConfigFactory.Avalonia/Builders/NumericControlBuilder.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Avalonia.Controls;
+using Avalonia.Data;
+using Avalonia.Layout;
+using ConfigFactory.Avalonia.Helpers;
+using ConfigFactory.Core;
+using ConfigFactory.Core.Attributes;
+using ConfigFactory.Generics;
+using System.Reflection;
+
+namespace ConfigFactory.Avalonia.Builders;
+
+public class NumericControlBuilder : ControlBuilder<NumericControlBuilder>
+{
+    public override object? Build(IConfigModule context, PropertyInfo propertyInfo)
+    {
+        // build as drop down
+        if (propertyInfo.GetCustomAttribute<DropdownConfigAttribute>() is DropdownConfigAttribute dca)
+        {
+            return DropdownBuilder.Build(context, propertyInfo, dca);
+        }
+
+        // build as numeric control
+        var control = new NumericUpDown
+        {
```

Comment:
More variables that should be statically typed. The variable name is also not great, but I'm not going to change it cause it'll mess everything else up.

```suggestion
        NumericUpDown control = new() {
```